src/dev/run_licenses_csv_report.js

```diff
@@ -71,7 +71,8 @@ run(
         licenses: [
           'Custom;https://www.redhat.com/licenses/EULA_Red_Hat_Universal_Base_Image_English_20190422.pdf',
         ],
-        sourceURL: 'https://oss-dependencies.elastic.co/redhat/ubi/ubi-minimal-8-source.tar.gz',
+        sourceURL: 
+          'https://oss-dependencies.elastic.co/red-hat-universal-base-image-minimal/8/ubi-minimal-8-source.tar.gz',
```

Comment:
```suggestion
         'https://oss-dependencies.elastic.co/red-hat-universal-base-image-minimal/8/ubi-minimal-8-source.tar.gz',
```

Apologies :) off by one

Reply:
Thanks for helping with the linter 👍🏻